broker/broker.go

```diff
@@ -278,6 +284,73 @@ func ValidateUserCreds(token string) error {
 	return nil
 }
 
+func CreateInternalStream(name string, subjects []string) error {
+	dedupWindow := time.Duration(1) * time.Nanosecond
+	_, err := js.AddStream(&nats.StreamConfig{
+		Name: "$memphis_" + name,
+		Subjects: subjects,
+		Retention: nats.WorkQueuePolicy,
+		MaxConsumers: -1,
+		Storage: nats.FileStorage,
+		Replicas: 1,
+		NoAck: false,
+		Duplicates: dedupWindow,
+	}, nats.MaxWait(10*time.Second))
+	if err != nil {
+		if !strings.Contains(err.Error(), "invalid bucket name"){
+			return getErrorWithoutNats(err)
```

Comment:
this if statement should check whether a Stream with that name is already exist and if it does => ignore this error, not return an error